docs/nlu/training-data-format.rst

```diff
@@ -30,6 +30,68 @@ e.g. ``[entity](entity name)``.
 
 .. code-block:: md
 
+Here is the example about how to get intent of user and how to get multiple entities by using RASA lookup tables and pre-     defined entities list for specific business use cases in two simple steps 
```

Comment:
Rather than improve the description of lookup tables here, it would be better to do this below.